client/src/components/floatingButton/floatingButton.tsx

```diff
@@ -24,23 +24,23 @@ function FloatingButton(): JSX.Element {
       content={
         <Menu>
           <MenuItem
-            href="https://github.com/chanzuckerberg/cellxgene-documentation/blob/main/README.md"
+            href="https://cellxgene.dev.single-cell.czi.technology/docs"
```

Comment:
This url is hardcoded to dev, we'll want this url to change as we go from deployment to deployment. So it's probably best to make it dynamic based on the data portal home page link passed here: https://github.com/chanzuckerberg/single-cell-explorer/blob/1b10f5ed62715ee3ecce1a352765c3a069f4db5d/client/src/globals.ts#L31-L47

Reply:
Just pushed some changes for this